### PR TITLE
API support

### PR DESCRIPTION
**Pull request by pokamest**

### CODE REVIEW

File: client/vpnconnection.cpp [hunk 1]
```diff
@@ -10,7 +10,7 @@
 #include <configurators/shadowsocks_configurator.h>
 #include <configurators/vpn_configurator.h>
 #include <configurators/wireguard_configurator.h>
-#include <core/servercontroller.h>
+#include "core/controllers/serverController.h"
```
Comment (tiaga) [inline client/vpnconnection.cpp:13]: Seems this typo leads to failed build and `<` `>` should be used.

File: client/protocols/openvpnprotocol.cpp [hunk 2]
```diff
@@ -214,7 +214,7 @@ ErrorCode OpenVpnProtocol::start()
     m_openVpnProcess->setProgram(PermittedProcess::OpenVPN);
     QStringList arguments({
             "--config", configPath(), "--management", m_managementHost, QString::number(mgmtPort),
-            "--management-client" /*, "--log", vpnLogFileNamePath */
+            "--management-client" , "--log", "/Users/nethius/Documents/git/log"
```
Comment (useribs) [inline client/protocols/openvpnprotocol.cpp:217]: Looks suspicious, by mistake?

File: client/ui/controllers/importController.cpp [hunk 3]
```diff
@@ -165,7 +167,11 @@ QJsonObject ImportController::extractAmneziaConfig(QString &data)
         ba = ba_uncompressed;
     }
 
-    return QJsonDocument::fromJson(ba).object();
+    QJsonObject config = QJsonDocument::fromJson(ba).object();
+
+    qDebug() << config;
```
Comment (outspace) [inline client/ui/controllers/importController.cpp:172]: please don't write config into the log file, it can contain sensitive data

**Review by Noxius87 — CHANGES_REQUESTED** [review]
Protocols are not switched in the protocol selection menu when switching from the main screen

**Comment by Nethius** [comment]
> Protocols are not switched in the protocol selection menu when switching from the main screen

fixed

**Review by Noxius87 — APPROVED** [review]

**Pull request merged**